.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 npm-debug.log
 conf.ts
 .env
-./dist
+dist
+package-lock.json
```

Comment:
I didn't put it in the docks yet, but can you use `yarn` instead of `npm i`?

Reply:
i will check out yarn

Reply:
The point of the lock file is so that everyone has consistent builds with the same package versions.